application.py

```diff
@@ -1229,5 +1222,331 @@ def inaccessible_callback(self, name, **kwargs):
 admin.add_view(UserModelView(User, db.session))
 
 
+
+# dash layout
+def fig_update(file_path):
+    """
+    Given the file path, return an updated fig graph to display.
+    :param file_path: string, to get csv file.
+    :return: fig, the styled graph.
+    """
+
+    cr_fig, sr_rolling, pnl_hist, pnl_df = None, None, None, None
+    logger.info(file_path)
+    if file_path is not None:
+        split_path = file_path.split('/')
+        prefix = "/".join(split_path[3:])
+
+        bucket_name = application.config["S3_BUCKET"]
+        csv_obj = s3_client.get_object(Bucket=bucket_name, Key=prefix)
+        pnl_df = pd.read_csv(csv_obj['Body'])
+
+        pnl_df['cusum'] = pnl_df['pnl'].cumsum()
+        cr_fig = px.line(pnl_df, x='date', y='cusum')
+
+        cr_fig.update_xaxes(
+            rangeslider_visible=True,
+            rangeselector=dict(
+                buttons=list([
+                    dict(count=1, label="1m", step="month", stepmode="backward"),
+                    dict(count=6, label="6m", step="month", stepmode="backward"),
+                    dict(count=1, label="YTD", step="year", stepmode="todate"),
+                    dict(count=1, label="1y", step="year", stepmode="backward"),
+                    dict(step="all")
+                ])
+            )
+        )
+
+        # Rolling sharpe ratio plot
+        pnl_df['rolling_SR'] = pnl_df.pnl.rolling(180).apply(lambda x: (x.mean() - 0.02) / x.std(), raw=True)
```

Comment:
why 180?

Reply:
what is 0.02?

Reply:
2% = expected banks saving interest rate. 
180 days = 30 * 6, 6 month. Industry standard.